### PR TITLE
June patch release notes

### PR DESCRIPTION
**Pull request by usha-mandya**
Adding DTR release notes for June patch release

### CODE REVIEW

**Comment by ollypom** [comment]
@usha-mandya Just pushed the offline bundles and updates to `_config.yml` for both UCP and DTR for the June Patch :)

File: _data/ddc_offline_files_2.yaml [hunk 1]
```diff
@@ -83,6 +93,16 @@
 - product: "ucp"
   version: "3.0"
   tar-files:
+    - description: "3.0.12 Linux"
+      url: https://packages.docker.com/caas/ucp_images_3.0.12.tar.gz
+    - description: "3.0.12 IBM Z"
+      url: https://packages.docker.com/caas/ucp_images_s390x_3.0.12.tar.gz
+    - description: "3.0.12 Windows Server 2016 LTSC"
+      url: https://packages.docker.com/caas/ucp_images_win_2016_3.0.12.tar.gz
+    - description: "3.0.12 Windows Server 1709"
```
Comment (ry4nz) [inline _data/ddc_offline_files_2.yaml]: Remove 1709

File: _data/ddc_offline_files_2.yaml [hunk 2]
```diff
@@ -5,7 +5,17 @@
 # Used by _includes/components/ddc_url_list_2.html
 - product: "ucp"
   version: "3.1"
-  tar-files: 
+  tar-files:
+    - description: "3.1.8 Linux"
+      url: https://packages.docker.com/caas/ucp_images_3.1.8.tar.gz
+    - description: "3.1.8 Windows Server 2016 LTSC"
+      url: https://packages.docker.com/caas/ucp_images_win_2016_3.1.8.tar.gz
+    - description: "3.1.8 Windows Server 1709"
```
Comment (ry4nz) [inline _data/ddc_offline_files_2.yaml]: Remove 1709

File: _data/ddc_offline_files_2.yaml (hunk 1 above)
Comment (usha-mandya) [reply _data/ddc_offline_files_2.yaml]: @ollypom is handling the offline bundle updates. Olly could you please remove 1709? Thanks.

Comment (ollypom) [reply _data/ddc_offline_files_2.yaml]: Thanks. Resolved in latest push

File: engine/release-notes.md [hunk 3]
```diff
@@ -29,6 +29,31 @@ consistency and compatibility reasons.
...
+### Builder
+
+* Fixed a panic error when building dockerfiles that contain only comments. [moby/moby#38487](https://github.com/moby/moby/pull/38487)
+* Added a workaround for GCR authentication issue. [moby/moby#38246](https://github.com/moby/moby/pull/38246)
+* Builder-next: Fixed a bug in the GCR token cache implementation workaround. [moby/moby#39183](https://github.com/moby/moby/pull/39183)
+
+### Runtime
+
+* Added performance optimizations in aufs and layer store that helps in the creation and removal of massively parallel containers. [moby/moby#39107](https://github.com/moby/moby/pull/39107)
+* Updated containerd to version 1.2.6. [moby/moby#39016](https://github.com/moby/moby/pull/39016)
```
Comment (kolyshkin) [inline engine/release-notes.md]: Hmm, it's not containers that are massively parallel, it's their creation and removal...

File: engine/release-notes.md [hunk 4]
```diff
@@ -29,6 +29,31 @@ consistency and compatibility reasons.
...
+
+### Builder
+
+* Fixed a panic error when building dockerfiles that contain only comments. [moby/moby#38487](https://github.com/moby/moby/pull/38487)
+* Added a workaround for GCR authentication issue. [moby/moby#38246](https://github.com/moby/moby/pull/38246)
+* Builder-next: Fixed a bug in the GCR token cache implementation workaround. [moby/moby#39183](https://github.com/moby/moby/pull/39183)
+
+### Runtime
+
+* Added performance optimizations in aufs and layer store that helps in the creation and removal of massively parallel containers. [moby/moby#39107](https://github.com/moby/moby/pull/39107)
```
Comment (kolyshkin) [inline engine/release-notes.md]: Hmm, it's not containers that are massively parallel, it's their creation and removal. Also, added layer store optimizations PR

```suggestion
* Added performance optimizations in aufs and layer store that helps in massively parallel container creation and removal. [moby/moby#39107](https://github.com/moby/moby/pull/39107), [moby/moby#39135](https://github.com/moby/moby/pull/39135)
```

**Pull request merged**